Feat/get rid of EXTRA=1 and MCL=1 (#106)
manticoresoftware/docker: component-licenses/README.MD

```
**License:** [GPL v3 or later](https://raw.githubusercontent.com/manticoresoftware/manticoresearch/master/LICENSE)

## Manticore Columnar Library

**Source:** [GitHub Repository](https://github.com/manticoresoftware/columnar)

Manticore Search can leverage the **Manticore Columnar Library** to enhance storage efficiency and query performance. The library is dynamically loaded at runtime

**License:** [Apache 2.0](https://raw.githubusercontent.com/manticoresoftware/columnar/master/LICENSE)

---

## Manticore Executor

**Source:** [GitHub Repository](https://github.com/manticoresoftware/executor)

Manticore Executor is a custom PHP build utilized by Manticore Buddy for query processing.

**License:** Distributed under the PHP 3.01 license, as included in the [manticore-executor-license](./manticore-executor-license) file.

---

## Manticore Search Backup

**Source:** [GitHub Repository](https://github.com/manticoresoftware/manticoresearch-backup)

Manticore Search Backup is a tool used for managing Manticore Search backups.

**License:** Distributed under the GPLv3 open-source license, included in the [manticoresearch-backup-license](./manticoresearch-backup-license) file.

---

> ### Manticore Search Buddy

Reviewer: ```suggestion
## Manticore Search Buddy
```

# Component Licenses

The **Docker image for Manticore Search** includes multiple third-party components, each distributed under its respective open-source license. This document provides an overview of these components, their sources, and licensing details.

## Manticore Search

**Website:** [Manticore Search](https://manticoresearch.com)  
**Source:** [GitHub Repository](https://github.com/manticoresoftware/manticoresearch)

**Copyright:**
```plaintext
Copyright (c) 2020-2024, Manticore Software LTD
All rights reserved.
```
**License:** [GPL v3 or later](https://raw.githubusercontent.com/manticoresoftware/manticoresearch/master/LICENSE)

> 
> ## Components Used in Manticore Search

Reviewer: ```suggestion
```

> ### Manticore Columnar Library

Reviewer: ```suggestion
## Manticore Columnar Library
```

**Website:** [Manticore Columnar Library](https://github.com/manticoresoftware/columnar)  
**Source:** [GitHub Repository](https://github.com/manticoresoftware/columnar)

Manticore Search uses the **Manticore Columnar Library** for improved storage efficiency and query performance. The library is utilized during compilation and dynamically loaded at runtime.

**License:** [Apache 2.0](https://raw.githubusercontent.com/manticoresoftware/columnar/master/LICENSE)

---

> ### Manticore Executor

Reviewer: ```suggestion
## Manticore Executor
```

> **Source:** Internal Manticore component

Reviewer: ```suggestion
**Source:** [GitHub Repository](https://github.com/manticoresoftware/executor)
```

> Manticore Executor is an execution engine used for query processing within Manticore Search.

Reviewer: ```suggestion
Manticore Executor is a custom PHP build utilized by Manticore Buddy for query processing.
```

> **License:** Distributed under an open-source license, included in the [manticore-executor-license](./manticore-executor-license) file.

Reviewer: ```suggestion
**License:** Distributed under the PHP 3.01 license, as included in the [manticore-executor-license](./manticore-executor-license) file.
```

---

> ### Manticore Search Backup

Reviewer: ```suggestion
## Manticore Search Backup
```

**Source:** Internal Manticore component

> Manticore Search Backup is a tool used for managing data backups within Manticore Search.
> 
> **License:** Distributed under an open-source license, included in the [manticoresearch-backup-license](./manticoresearch-backup-license) file.

Reviewer: ```suggestion
Manticore Search Backup is a tool used for managing Manticore Search backups.

**License:** Distributed under the GPLv3 open-source license, included in the [manticoresearch-backup-license](./manticoresearch-backup-license) file.
```

---

### Manticore Search Buddy

> **Source:** Internal Manticore component

Reviewer: ```suggestion
**Source:** [GitHub Repository](https://github.com/manticoresoftware/manticoresearch-buddy)
```

> Manticore Search Buddy is a management and optimization utility for Manticore Search instances.

Reviewer: ```suggestion
Manticore Buddy is a sidecar for Manticore Search, written in PHP, which helps with various tasks.
```

> **License:** Distributed under an open-source license, included in the [manticoresearch-buddy-license](./manticoresearch-buddy-license) file.

Reviewer: ```suggestion
**License:** Distributed under the open-source GPLv3 license, included in the [manticoresearch-buddy-license](./manticoresearch-buddy-license) file.
```

---

> 
> ## License Compliance and Updates
> 
> 1. **Adding a New Dependency**
>     - If a new third-party dependency is added to the Docker image, its license must be retrieved and placed in this directory.
>     - The file should be named appropriately, e.g., `new-component-license`.
> 
> 2. **Updating Licenses**
>     - When a component updates its licensing terms, the corresponding license file in this directory should be updated.
> 
> 3. **Compliance**
>     - This directory is included in the Docker image build to maintain legal compliance with open-source licenses.

Reviewer: ```suggestion
```
